### PR TITLE
Faranosh_Amini Portfolio

### PR DESCRIPTION
**Pull request by FaranoshAmini**
In this Pull Request I completed the following tasks:
- Used the [HTML&CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css)
- Built the  2 following sections on my project:
    - the toolbar (or header):
    - the headline section (right after the header):

- Used Flexbox in all 2 sections in order to layout the elements on the page.
- Stick to the design as much as possible (e.g. font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+.headline {
+  width: 375px;
+  height: 718px;
+  background-image: url("./Images/header-shapes mobile.png");
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 70px 24px;
+}
```
Comment (tresorsawasawa) [inline style.css:35]: - [ ] Kindly make sure your `background image` covers the entire page to match the Figma design and to make your pages look smart. I suggest you change the `width: 375px;` to `width: 100vw;` and add add the `background-size: 100% or cover;`. After that, you can use the correct given properties by Figma design to match, such as: `size`, `margin`, `paddings`, `fonts`,`etc.`
- Figma design:
Figma_1
- Your design:
now

File: style.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  margin-top: 58px;
+}
+
+.logo {
+  width: 76px;
+  height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
```
Comment (tresorsawasawa) [inline style.css:15]: - [ ] Kindly add on the **logo text** the correct `font-size`, `font-weight`, `font-family`, `etc` provided from **Figma** to make your design look smart.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @FaranoshAmini 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Require changes :recycle::recycle: 
- [x] Kindly make your project's design stick as well as possible to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) design by adjusting the correct `font`, `colors`, `images`, `paddings`, `margins`, `size`, `etc`.
- [x] I suggest you delete the text starting with `**Modify this file to match your proj...` as they have no importance in this documentation and will make your documentation clean.
- [x] **Check other comments under the review**.

## Optional suggestions
- [x] - [ **OPTIONAL** ]  I think it would be better :100: if you add some hints on how to **clone** the projects and **access** the **cloned directory** and run the **app locally** and the `Prerequisites` tools` like **Vscode**, **Git**, etc. and sections to make your Readme more professional.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  color: #7f8cff;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  margin-bottom: 20px;
+}
+
+.social-media-icons {
```
Comment (topeogunleye) [inline style.css:80]: - [ ] As suggested by the previous code reviewer, I kindly suggest that you correct the margin from the bottom of "LETS CONNECT" to your social media icons

| Your Design  | Template Design |
| ------------- | ------------- |
| Screenshot from 2022-05-27 10-40-42  | Screenshot from 2022-05-27 10-41-04  |

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @FaranoshAmini,
> Nice job, you are almost there.
> 
>Screenshot from 2022-05-27 10-25-07
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
>- [ ] From now on, I kindly suggest that you please make use of imperative mood in the subject line for your commit messages.
:x: Wrong
Screenshot from 2022-04-20 08-55-14
> ### Good Commit Messages:
>- [x]  Add fix for dark mode toggle state
>- [x] Update README file.
>- [x] Add styles to h1.
>

> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css (hunk 2 above)
Comment (FaranoshAmini) [reply style.css:15]: Thank you so much.

File: style.css (hunk 3 above)
Comment (FaranoshAmini) [reply style.css:80]: Thank you so much

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @FaranoshAmini 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Excellent job implementing changes requested by previous reviewers 😎 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**